packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.11.0
```

Comment:
This should only be bumped to 0.10.1 since it's a non breaking change on a pre-1.0 plugin.

Reply:
I have add your suggested change, but I would like to point out that this PR moves the CameraDescription from the CameraController into the CameraController's CameraValue which I was assuming was a breaking change and want to be sure it is not?

Reply:
Why not just add a forwarding getter to the controller so that the API of the controller doesn't change?

Reply:
That is perfect. I added the getter and updated the versioning on camera_avfoundation and camera_android as well since its no longer a breaking change. Would you be able to verify those versions are correct?